### PR TITLE
Add SurrealDB

### PR DESCRIPTION
**Pull request by Ismaaa**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,325 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
```
Comment (githubsaturn) [inline yarn.lock:3]: Please remove this file.

**Comment by githubsaturn** [comment]
Looks good! Just one minor comment.

File: yarn.lock (hunk 1 above)
Comment (Ismaaa) [reply yarn.lock]: Done 👍

**Comment by githubsaturn** [comment]
Thanks!

**Comment by MattPhantastic** [comment]
SurrealDB crashes on start with the following error message and is unusable. How or where do I open an issue?

```
ERROR surreal::cli: There was a problem with the database: There was a problem with a datastore transaction: Failed to create RocksDB directory: `Os { code: 13, kind: PermissionDenied, message: "Permission denied" }`.
```

**Pull request merged**